### ISSUE TITLE
Some colors are not matching the Corporate Design Manual

### ISSUE THREAD

**Issue by keszocze** [FAU-AMMN/fau-beamer#28]
The Corporate Design Manual (Stand 01.2024) on page 42 defines the colors of the design. It seems that the FAU switched colors. I could retrieve the old title page image for the TechFak from somewhere (see pull request #27) but do not have the title images for the other faculties, hence this issue.

The correct colors should be
```LaTeX
% ..............................................................................
% Define the colors and their respective dark versions for each institute.
% ------------------------------------------------------------------------------

% FAU University colors
\definecolor{FAUBlue}{RGB}{0,47,108}
\definecolor{FAUDarkBlue}{RGB}{4,30,66}

% Phil colors
\definecolor{PhilYellow}{RGB}{255, 184, 28}
\definecolor{PhilOrange}{RGB}{232, 119, 34}

% RW colors
\definecolor{RWRed}{RGB}{200, 16, 46}
\definecolor{RWDarkRed}{RGB}{151,27,47}

% Med colors
\definecolor{MedBlue}{RGB}{0, 163, 224}
\definecolor{MedDarkBlue}{RGB}{0, 97, 160}

% Nat colors
\definecolor{NatGreen}{RGB}{67, 176, 42}
\definecolor{NatDarkGreen}{RGB}{34, 136, 72}

% Tech colors
\definecolor{TechMetallic}{RGB}{140, 159, 177}
\definecolor{TechDarkMetallic}{RGB}{47, 88, 110}
```

You can just copy and paste them but still need to retrieve the correct title images. I *believe* that they might be in some old commit but I did not have the time to search for them.

**Comment by TimRoith** [comment]
Hi sorry for just accepting the PR without properly reviewing it. I now remember the issue, which dates back to 2021, when the design manual was created: The RGB values defined on page 44 do not fit the images that depict them, correct me if I'm wrong but a quick check with a color picker, confirms that they still differ. I had a lot of phone calls with people from corporate design in 2021 and we agreed to choose the colors that are depicted.

If you agree that the previous color version match the colors depicted in the manual, I would revert the PR.

**Comment by TimRoith** [comment]
PS: It is mainly Med, Nat and TF that differ.

**Comment by keszocze** [comment]
> Hi sorry for just accepting the PR without properly reviewing it. I now remember the issue, which dates back to 2021, when the design manual was created: The RGB values defined on page 44 do not fit the images that depict them, correct me if I'm wrong but a quick check with a color picker, confirms that they still differ. I had a lot of phone calls with people from corporate design in 2021 and we agreed to choose the colors that are depicted.

Are you sure about that? I downloaded the `PP-Master-FAU Tech 16zu9.pptx` (Stand 19.07.2022) and the color there looks identical to the one you get from my PR. I did *not* check other templates.

If you open the presentation you see that they actually changed the template *from* the colors you had in your template to the ones from my PR. They did it by adding an overlay over the colors I replaced

**Comment by TimRoith** [comment]
In the pptx it uses the reported RGB value that is true, however this is not the color that is depicted in the design manual

<img width="410" alt="image" src="https://github.com/FAU-AMMN/fau-beamer/assets/44805883/67b4f5a0-940f-43c0-a2c4-7ef2bb388eb5">

The left is from the pptx, which gives me the RGB reported on the right. However, the image on the right is from the manual page 44, and as far as I understood the values depicted there are the ones that count.

This in particular relevant for myself, because initially there were a lot of complaints about the new Nat green being much too bright for presentations, that's why a different green was used, the one depicted in the manual.

**Comment by keszocze** [comment]
So basically the official manual has RGB values that **do not** match the example bars above them?  I, of course, did not check for that! 

Furthermore they are even using the RGB values  in their own PP templates? This is super crazy.

**Comment by TimRoith** [comment]
It is a mistake which came due to changes of the color scheme, when it was introduced. After the last change, I agreed with a corresponding person from corporate design, that the colors should match the visual appearance in the style guide. 

For Med, Nat and TF, basically one can either choose the values or the pictures as reference, leading to different color schemes. For Nat the issue with the green given by the values is, that it is much too bright. To be consistent, I therefore decided to define the pictures as the correct colors. The PP template either copied the current values or was not updated after the last color change.

**Comment by TimRoith** [comment]
I guess I could add the options to use the RGB specified color schemes. For now I'll revert the PR to have a consistent choice again. I'll leave the issue open until I added the second color option for Med, Nat and Tf.

**Comment by keszocze** [comment]
That seems to be a reasonably choice.

Btw, do you think that it makes sense to send them an email and ask for clarification? The website states that per some decree using the corporate design is now mandatory.

**Comment by TimRoith** [comment]
Yes, I can contact them again. I repeatedly pointed out the issue in 2021, but since then some time has passed and maybe now there is some motivation to fix it.

**Comment by TimRoith** [comment]
So after more than two months, and some discussions via mail, I did not receive a meaningful reply and at this point I will stop trying. 

I guess the best option is to go ahead and offer both color schemes. However, unfortunately, I am not affiliated with FAU anymore, therefore I can not access the style guide.  

If I remember correctly, the RGB values floating in some older commit, are the ones that are currently specified in the style guide. If someone with access could confirm this, then I will reintroduce them as color options. One issue, is that I'm not too sure about the title images (if the old ones are not there), they have to be produced separately. 

The process is as follows: In PowerPoint there is a set of these title images in different colors, not matching the theme. Then a color gradient is layered over the image to make it fit the scheme. So far, I refrained from using tikz or something similar to replicate this process, but instead do this beforehand with a small python script (which is not pushed).

So that's the only ToDo, that requires more work, if the old title images are not there anymore.

**Comment by TimRoith** [comment]
An additional problem I just realized: the logo colors will also not match the scheme. One could upload all of them, but there are two cons:

* the logo usage is quite non-permissive, I only got the explicit permit to upload the files that are here right now. For new logo files, there would probably be a new process, which I am not willing to do.
* the repo size would increase, however there are already a lot of old .png, .jpg in the history so probably this would not be too bad.

I think in this case the logo files will default to the color scheme that is here right now and then the user will have to replace them for a matching scheme.

**Comment by keszocze** [comment]
I also moved to another university and, hence, do not care too much anymore. It just is quite weird that there is not much interest from the officials to get this done right after publishing such a long document.
